src/functionalTest/resources/scenarios/RIA-5445-derive-hearing-center-from-wa-dummy-postcode.json

```diff
@@ -0,0 +1,48 @@
+{
+  "description": "RIA-5445 Derive hearing centre from dummy postcode for Work Allocation team's functional tests",
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "LegalRepresentative",
```

Comment:
I can't see this file being used anywhere in the PR, where is it being used?

Reply:
It is a scenario in the functional test suite. It would be used when the functional test is run.